### PR TITLE
null check cleanup

### PR DESCRIPTION
**Pull request by pq**
Another round of de-null-checking.

/cc @bwilkerson

### CODE REVIEW

File: lib/src/rules/avoid_relative_lib_imports.dart [hunk 1]
```diff
@@ -64,9 +64,12 @@ class _Visitor extends SimpleAstVisitor<void> {
     // resolved URI and not it's literal string content.
     // See: https://github.com/dart-lang/linter/issues/2419
     try {
-      final uri = Uri.parse(node.uriContent!);
-      if (uri.scheme.isEmpty) {
-        return uri.path.contains('/lib/');
+      var uriContent = node.uriContent;
+      if (uriContent != null) {
+        final uri = Uri.parse(uriContent);
```
Comment (parlough) [inline lib/src/rules/avoid_relative_lib_imports.dart:69]: Same here.

**Review by parlough — APPROVED** [review]

File: lib/src/rules/always_use_package_imports.dart [hunk 2]
```diff
@@ -74,8 +74,11 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   bool isRelativeImport(ImportDirective node) {
     try {
-      final uri = Uri.parse(node.uriContent!);
-      return uri.scheme.isEmpty;
+      var uriContent = node.uriContent;
+      if (uriContent != null) {
+        final uri = Uri.parse(uriContent);
```
[the remark this answers is not in the document]
Comment (pq) [reply lib/src/rules/always_use_package_imports.dart:79]: Nice one!  Great idea.

File: lib/src/rules/avoid_relative_lib_imports.dart (hunk 1 above)
Comment (pq) [reply lib/src/rules/avoid_relative_lib_imports.dart:69]: 🔥

File: lib/src/rules/cascade_invocations.dart [hunk 3]
```diff
@@ -128,20 +128,19 @@ class CascadeInvocations extends LintRule implements NodeLintRule {
...
   /// Whether this expression can be joined with a previous expression via a
   /// cascade operation.
-  final bool? canJoin;
+  final bool canJoin;
 
   /// Whether this expression can receive an additional expression with a
   /// cascade operation.
   ///
   /// For example, `a.b = 1` can receive, but `a = 1` cannot receive.
   final bool? canReceive;
```
Comment (bwilkerson) [inline lib/src/rules/cascade_invocations.dart:142]: Hopefully we can get rid of this one eventually too.

**Review by bwilkerson — APPROVED** [review]

File: lib/src/rules/cascade_invocations.dart (hunk 3 above)
Comment (pq) [reply lib/src/rules/cascade_invocations.dart:142]: Ah.  I missed that one.  Fixed now.  Thanks!

**Pull request merged**